osdk/tests/examples_in_book/write_a_kernel_in_100_lines_templates/lib.rs

```diff
@@ -28,12 +28,20 @@ use ostd::user::{ReturnReason, UserMode, UserSpace};
 #[ostd::main]
 pub fn main() {
     let program_binary = include_bytes!("../hello");
-    let user_space = create_user_space(program_binary);
-    let user_task = create_user_task(Arc::new(user_space));
+    let (user_space, user_ctx) = create_user_space_and_context(program_binary);
+    let user_task = create_user_task(Arc::new(user_space), Arc::new(user_ctx));
+    // Injects a post-schedule handler to activate the user-space when switching to it.
+    ostd::task::inject_post_schedule_handler(post_schedule_handler);
```

Comment:
I suggest that we remove this injection method. Our Hello World kernel can work without it. Just invoking `UserSpace::activate` once is enough.  Let's try to keep the learning curve as flat as possible in this sample project.